client/src/components/MocUser.js

```diff
@@ -0,0 +1,31 @@
+// Fake contact data for testing purposes.
+
+const MocUser = [
```

Comment:
Try and make names plural if they can contain multiple elements. Typically for array and objects.